html/html-spec.md

```diff
@@ -339,8 +249,29 @@ Result (lab started, same as the previous example):
 
 ![example preview](./images/ql-variable-03.png)
 
-### `<ql-activity-tracking>`
+## `<ql-hint>`
```

Comment:
I think these changes should happen in google3, then Copybara will copy it over to here. https://source.corp.google.com/piper///depot/google3/cloud/training/qwiklabs/content_bundle/g3doc/html/html-spec.md